src/modules/pages/NewOAuthServerPage/OAuthServerForm.js

```diff
@@ -0,0 +1,446 @@
+                                            value={'true'}
+                                            type="radio"
+                                            id="rate-limit-is-active"
+                                        />
+                                        <Label htmlFor="rate-limit-is-active">Yes</Label>
+                                    </Row>
+                                    <Row className={b('radio')()}>
+                                        <Field
+                                            name="rate_limit.enabled"
+                                            component={Radio}
+                                            value={'false'}
+                                            type="radio"
+                                            id="rate-limit-is-not-active"
+                                        />
+                                        <Label htmlFor="rate-limit-is-not-active">No</Label>
+                                    </Row>
+                                </Row>
+                            </div>
+                        </div>
+                    </div>
+                    <div className={b('section')}>
+                        <div className={b('section-title')}>4. Token strategy</div>
+                        <div className={row({fullwidth: true}).mix('j-api-form__row')}>
+                            <div className={row('item')}>
+                                <Select
+                                    className="j-select"
+                                    name="token_strategy.name"
+                                    options={createStrategyOptions(schema.token_strategy.strategies)}
+                                    onChange={handleChangeStrategy}
+                                    value={this.state.strategy.name}
+                                    searchable={false}
+                                    clearable={false}
+                                />
+                            </div>
+                        </div>
+                        { renderStrategy(this.state.strategy.name) }
+                    </div>
+                </div>
+
+                <div className={b('inner', {overflowed: true})}>
+                    <div className={b('section')}>
+                        { renderTabs() }
+                    </div>
+                </div>
+
+                <Row className={b('row',{ 'button-row': true })()}>
+                    <Button
+                        type="submit"
+                        mod="primary"
+                    >
+                        Create oAuth Server
+                    </Button>
+                </Row>
+            </form>
+        );
+    }
+};
+
+OAuthServerForm.propTypes = propTypes;
+
+const selector = formValueSelector('oAuthServerForm');
+
+const form = reduxForm({
+    form: 'oAuthServerForm',
+    enableReinitialize: true, // this is needed!!
+})(OAuthServerForm);
+
+export default connect(
+    // state => {
```

Comment:
🤔

Reply:
yeap, sorry 😞  removed. Redundant as a connect at this moment.

Reply:
I didn't follow sorry, what you meant?